rhel/SPECS/pmm-server.spec

```diff
@@ -42,9 +42,6 @@ install -d %{buildroot}%{_datadir}/percona-dashboards
 
 mv alertmanager.yml %{buildroot}%{_sysconfdir}/alertmanager.yml
 
-install -d %{buildroot}%{_sysconfdir}/supervisord.d
-mv supervisord.conf %{buildroot}%{_sysconfdir}/supervisord.d/pmm.ini
-
```

Comment:
I moved `supervisord.conf`. Because of two reasons:
1. We can more flexibly manage WHEN we want to replace file.
2. We want move all config to Ansible.